desktop/app/pro.go

```diff
@@ -86,17 +86,17 @@ func (app *App) IsProUser(ctx context.Context) (isPro bool, ok bool) {
 	if err != nil {
 		return false, false
 	}
-	return IsProUser(ctx, app.proClient, app.settings.GetUserID())
+	return IsProUser(ctx, app.proClient, app.settings)
 }
 
-func IsProUser(ctx context.Context, proClient pro.ProClient, userId int64) (isPro bool, ok bool) {
+func IsProUser(ctx context.Context, proClient pro.ProClient, uc common.UserConfig) (isPro bool, ok bool) {
 	isActive := func(user *protos.User) bool {
 		return user != nil && user.UserStatus == "active"
 	}
-	user, found := GetUserDataFast(ctx, userId)
+	user, found := GetUserDataFast(ctx, uc.GetUserID())
```

Comment:
This doesn't fix the issue, If the user is on the payment page, then we already have user data, which will always return local data. Still, what we want is if a user is on the payment page then we need to call userData API again and again till the user is on the page, SO we will know then user has made the payment outside the side app, On the System browser.